omnibus/Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 # Install omnibus software
 group :omnibus do
   gem 'omnibus', git: 'https://github.com/chef/omnibus'
-  gem 'omnibus-software', git: 'https://github.com/chef/omnibus-software'
+  gem 'omnibus-software', git: 'https://github.com/chef/omnibus-software', branch: 'csnapp/add_sqitch_0.9999'
```

Comment:
This commit needs to be reverted once PR https://github.com/chef/omnibus-software/pull/1304 is completed